src/ml/neural_net/style_transfer/mps_style_transfer_encoding_node.h

```diff
@@ -0,0 +1,24 @@
+#import <Accelerate/Accelerate.h>
+#import <Foundation/Foundation.h>
+#import <Metal/Metal.h>
+#import <MetalPerformanceShaders/MetalPerformanceShaders.h> 
+
+#include <ml/neural_net/style_transfer/mps_style_transfer_utils.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+API_AVAILABLE(macos(10.14))
+@interface TCMPSStyleTransferEncodingNode : NSObject 
+
+@property (nonatomic) MPSNNImageNode *m_output;
```

Comment:
Objective C properties don't get prefixes like `m_`. This should just be called `output`.

By default, you'll automatically get an instance variable named `_output`.